openapi/kas-fleet-manager.yaml

```diff
@@ -970,6 +1014,68 @@ components:
         plan:
           description: kafka plan in a format of <instance_type>.<size_id>
           type: string
+    SupportedKafkaInstanceTypesList:
+      allOf:
+        - type: object
+          example:
+            instance_types:
+              $ref: '#/components/examples/SupportedKafkaInstanceTypeListExample'
+          properties:
+            instance_types:
+              type: array
+              items:
+                allOf:
+                  - $ref: '#/components/schemas/SupportedKafkaInstanceType'
+    SupportedKafkaInstanceType:
+      description: 'Supported Kafka instance type'
+      type: object
+      properties:
+        id:
+          description: 'Unique identifier of the Kafka instance type.'
+          type: string
+        sizes:
+          description: ' A list of Kafka instance sizes available for this instance type'
+          type: array
+          items:
+            allOf:
+              - $ref: '#/components/schemas/SupportedKafkaSize'
+    SupportedKafkaSize:
+      description: 'Supported Kafka Size'
+      type: object
+      properties:
+        id:
+          description: 'Unique identifier of this Kafka instance size.'
+          type: string
+        ingress_throughput_per_sec:
+          description: 'Ingress throughput per second available to this Kafka instance size.'
+          type: string
+        egress_throughput_per_sec:
+          description: 'Egress throughput per second available to this Kafka instance size.'
+          type: string
+        total_max_connections:
+          description: 'Total amount of connections available to this Kafka instance size.'
+          type: integer
+        max_data_retention_size:
+          description: 'Maximum data storage available to this Kafka instance size.'
+          type: string
+        max_partitions:
+          description: 'Total amount of partitions available to this Kafka instance size.'
+          type: integer
+        max_data_retention_period:
+          description: 'Maximum data retention period available to this Kafka instance size.'
+          type: string
+        max_connection_attempts_per_sec:
+          description: 'Maximium connection attempts per second available to this Kafka instance size.'
+          type: integer
+        quota_consumed:
+          description: 'Quota consumed by this Kafka instance size.'
+          type: integer
+        quota_type:
+          description: 'Quota type used by this Kafka instance size.'
+          type: string
+        capacity_consumed:
+          description: 'Data plane cluster capacity consumed by this Kafka instance size.'
+          type: integer
```

Comment:
Do we need to return `capacity_consumed`? This is only used by KAS Fleet Manager and clients shouldn't need to know about it. 

Not too sure about `quota_consumed` and `quota_type` too.

Reply:
They're included in ADR-81 https://github.com/bf2fc6cc711aee1a0c2a/architecture/pull/42/files#diff-4a09f9937fd7e06026f7cfe0e7d4143d202388410486747c30750de03aca4a26R119-R121. Seems to be needed by UI

Reply:
thanks for confirming! :)